components/public-api-server/pkg/server/server.go

```diff
@@ -187,6 +218,12 @@ func register(srv *baseserver.Server, deps *registerDependencies) error {
 	// OIDC sign-in handlers
 	rootHandler.Mount("/oidc", oidc.Router(deps.oidcService))
 
+	idpRoute, idpServiceHandler := v1connect.NewIDPServiceHandler(apiv1.NewIDPService(deps.connPool, deps.idpService), handlerOptions...)
+	rootHandler.Mount(idpRoute, idpServiceHandler)
+
+	// IDP well-known and keys routes
+	rootHandler.Mount("/idp", deps.idpService.Router())
```

Comment:
Out of curiosity, what stops us serving these on the `idpServicehandler` rather than having to have another endpoint?